### ISSUE TITLE
should not remove comma within ES6 template literals

### ISSUE THREAD

**Issue by futurist** [maxogden/standard-format#178]
below code:

```javascript
    expect(ret.css).equal(
      `p {
display: -webkit-flexbox;
display: -moz-flexbox;
display: flex;
color: red;
}
`)
```

When run format, all comma removed.

This should check the ES6 template literals case.

**Comment by chenxsan** [comment]
I would suggest running `standard --fix` (added since v8.0.0) if your codes are written in ES6/ES7/ES8. It works much better than standard-format.

**Comment by futurist** [comment]
Oh, thank you for that!
Also, you can check my lib: https://github.com/cssobj/cssobj, hope it's useful!
